fk/luae_na/src/test/java/se/inera/intyg/common/luae_na/v1/model/converter/certificate/question/QuestionGrundForMUUnderlagTest.java

```diff
@@ -0,0 +1,302 @@
+                null, null, null, null);
+
+            final var firstValidation = (CertificateDataValidationMaxDate) question.getValidation()[0];
+            final var secondValidation = (CertificateDataValidationMaxDate) question.getValidation()[1];
+            final var thirdValidation = (CertificateDataValidationMaxDate) question.getValidation()[2];
+            final var fourthValidation = (CertificateDataValidationMaxDate) question.getValidation()[3];
+
+            assertAll(
+                () -> assertEquals(GRUNDFORMEDICINSKTUNDERLAG_UNDERSOKNING_AV_PATIENT_SVAR_JSON_ID_1, firstValidation.getId()),
+                () -> assertEquals(GRUNDFORMEDICINSKTUNDERLAG_JOURNALUPPGIFTER_SVAR_JSON_ID_1, secondValidation.getId()),
+                () -> assertEquals(GRUNDFORMEDICINSKTUNDERLAG_ANHORIGS_BESKRIVNING_SVAR_JSON_ID_1, thirdValidation.getId()),
+                () -> assertEquals(GRUNDFORMEDICINSKTUNDERLAG_ANNAT_SVAR_JSON_ID_1, fourthValidation.getId())
+            );
+        }
+
+        @Test
+        void shouldIncludeMaxDateValidationLimit() {
+            final var question = QuestionGrundForMUUnderlag.toCertificate(0, texts,
+                null, null, null, null);
+
+            final var firstValidation = (CertificateDataValidationMaxDate) question.getValidation()[0];
+            final var secondValidation = (CertificateDataValidationMaxDate) question.getValidation()[1];
+            final var thirdValidation = (CertificateDataValidationMaxDate) question.getValidation()[2];
+            final var fourthValidation = (CertificateDataValidationMaxDate) question.getValidation()[3];
+
+            final var expectedNumberOfDays = (short) 0;
+
+            assertAll(
+                () -> assertEquals(expectedNumberOfDays, firstValidation.getNumberOfDays()),
+                () -> assertEquals(expectedNumberOfDays, secondValidation.getNumberOfDays()),
+                () -> assertEquals(expectedNumberOfDays, thirdValidation.getNumberOfDays()),
+                () -> assertEquals(expectedNumberOfDays, fourthValidation.getNumberOfDays())
+            );
+        }
+
+        @Test
+        void shouldIncludeValidationMandatoryExpression() {
+            final var expectedExpression =
+                GRUNDFORMEDICINSKTUNDERLAG_UNDERSOKNING_AV_PATIENT_SVAR_JSON_ID_1 + " || "
+                    + GRUNDFORMEDICINSKTUNDERLAG_JOURNALUPPGIFTER_SVAR_JSON_ID_1 + " || "
+                    + GRUNDFORMEDICINSKTUNDERLAG_ANHORIGS_BESKRIVNING_SVAR_JSON_ID_1 + " || "
+                    + GRUNDFORMEDICINSKTUNDERLAG_ANNAT_SVAR_JSON_ID_1;
+            final var question = QuestionGrundForMUUnderlag.toCertificate(0, texts, null,
+                null, null, null);
+            final var mandatoryValidation = (CertificateDataValidationMandatory) question.getValidation()[4];
+
+            assertEquals(expectedExpression, mandatoryValidation.getExpression());
+        }
+    }
+
+    private static Stream<InternalDate> dateValues() {
```

Comment:
Wrap in a nested ToInternal to follow "unwritten" test pattern? Also, easier to keep tests together if more are needed.